### PR TITLE
Check Caliber alpha deployment

### PR DESCRIPTION
**Pull request by dmaklygin**
DO NOT MERGE

[FX-NNNN]

### Description

Describe the changes and motivations for the pull request, unless evident from the title.

### How to test

- FIXME: Add the steps describing how to verify your changes

### Development checks

- [ ] Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

### CODE REVIEW

File: build-publish-alpha-package/action.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ runs:
         --alpha \
         --outputVersion .version \
         --branch "$BRANCH" \
+        --publishRootFolder .
```
Comment (OndrejTuma) [inline build-publish-alpha-package/action.yml:51]: Can this be an input perhaps?

Comment (dmaklygin) [reply build-publish-alpha-package/action.yml:51]: yes, we should extend this GH Action, to let developers configure root folder.

**Comment by dmaklygin** [comment]
Close and remove branch after merging&releasing this [PR](https://github.com/toptal/davinci-github-actions/pull/127)

**Comment by dmaklygin** [comment]
Hey @vshyrokov, it this still needed?

**Comment by TomasSlama** [comment]
Caliber is using latest version now https://github.com/toptal/caliber/pull/319/files, we can safely close this PR

**Pull request closed without merging**